modules/ROOT/pages/_partials/commons/common-logging.adoc

```diff
@@ -29,94 +29,178 @@ include::{root-partials}_block-abstract.adoc[]
 endif::[]
 // End -- Output an abstract unless this inclusion is being used as part of an encompassing page
 
+:logging-classes: {url-api-class-log}, {url-api-class-log-file} and {url-api-class-log-config}
+ifeval::["{param-platform}"=="{platform-net}"]
+:logging-classes: {url-api-class-logging}
+endif::[]
+
+ifdef::is-android[]
+.Constraints
+endif::[]
+
+[NOTE]
+--
+The retrieval of logs from the device is out of scope of this feature.
+
+ifdef::is-android[]
+The value returned by `LogLevel.getValue()` is not the Android log level.
+Do not use this API call.
+endif::[]
+--
+
 
 // Begin text block
-Couchbase Lite provides a logging API that unifies the logging behavior across all platforms{fn-2x5}, making debugging and troubleshooting easier during development and in production.
+== Introduction
+Couchbase Lite provides a robust Logging API {fn-2x5} -- see: API References for {logging-classes} -- which make debugging and troubleshooting easier during development and in production.
+It delivers flexibility in terms of how logs are generated and retained, whilst also maintaining the level of logging required by Couchbase Support for investigation of issues.
+
+Log output is split into the following streams:
+
+* <<lbl-file-logs>>
++
+Here logs are written to <<log-file-outputs, separate log files>> filtered by log level, with each log level supporting individual retention policies.
+
+* <<lbl-console-logs>>
++
+--
+You can independently configure and control console logs, which provides a convenient method of accessing diagnostic information during debugging scenarios.
```

Comment:
Would it be worth noting, here, that, on Android, the most effective strategy is to turn file logging off, altogether (ignore the warning that this produces), and to use the Console logger (which logs to logcat) exclusively.

On the Android platform, we strongly prefer the output of the console log (logcat) to the file logs.

Reply:
added

`TIP: On Android, it can often be more effective to solely use the Console logger (which logs to logcat). +
Note, a warning is displayed when you set continuous (file) logging *off*.
`